Justfile

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env just --justfile
+set dotenv-load := true
+
+init env module:
+  #!/usr/bin/env bash
+  cd {{module}}
+  terraform init
+  terraform workspace list | grep -q "{{env}}"
+  if [ $? -eq 0 ]; then
+    echo "Workspace '{{env}}' already exists"
+  else
+    echo "Creating new Terraform workspace {{env}}"
+    terraform workspace new {{env}}
+  fi
+
+  key_name="{{env}}-testnet-infra"
+  if ! aws ec2 describe-key-pairs --key-names "$key_name" > /dev/null 2>&1; then
+    pub_key=$(cat $SSH_PUBLIC_KEY_PATH | base64 -w0 | xargs)
+    echo "Creating new key pair for the testnet infra..."
+    aws ec2 import-key-pair \
+      --key-name $key_name --public-key-material $pub_key
+  fi
+
+networking env: #!/usr/bin/env bash
```

Comment:
Found an issue with this particular line, where it always failed with the following error msg,
```
cd networking
terraform workspace select sherwin

Workspace "sherwin" doesn't exist.

You can create this workspace with the "new" subcommand.
error: Recipe `networking` failed on line 28 with exit code 1
```
Turns out, we should move `#!/usr/bin/env bash` to the next line. This fixes the issue.